frontend/packages/console-shared/src/components/dashboard/dashboard-card/card.scss

```diff
@@ -6,6 +6,20 @@ $co-dashboard-card--line-height: 19px;
   margin: 0;
 }
 
+.co-dashboard-card--gradient {
+  position: relative;
+  &:after {
+    background: linear-gradient(to bottom, #fff0 0%, #ffffffe6 100%);
```

Comment:
```suggestion
    background: linear-gradient(transparent, var(--pf-global--BackgroundColor--100));
```

`to bottom` is default - can be omitted, the percentage values define where the gradient starts rather than opacity, using background color var is a bit safer.

Reply:
fixed

Reply:
@jtomasek @rawagner did you test this change in all browsers?
Safari doesn't seem to like it for me :(
<img width="1133" alt="Screen Shot 2020-01-06 at 10 21 50 AM" src="https://user-images.githubusercontent.com/287880/71823769-7377fa80-306e-11ea-8384-59e6f5c5026d.png">

Reply:
To achieve transparency in Safari it needs to rgba value with alpha set to 0
`background: linear-gradient(rgba(255, 255, 255, 0), var(--pf-global--BackgroundColor--100))`

Reply:
sorry about that, here's a followup PR https://github.com/openshift/console/pull/3878